integration/deploy_test.go

```diff
@@ -38,43 +38,21 @@ func TestBuildDeploy(t *testing.T) {
 
 	ns, client := SetupNamespace(t)
 
-	outputBytes := skaffold.Build("--quiet", "--platform=linux/arm64,linux/amd64").InDir("examples/microservices").InNs(ns.Name).RunOrFailOutput(t)
+	outputBytes := skaffold.Build("--quiet", "--platform=linux/arm64,linux/amd64").InDir("examples/nodejs").InNs(ns.Name).RunOrFailOutput(t)
```

Comment:
nit:  in the PR description you mention:
```
refactor tests builds cross-platform/multi-platform examples/microservice project to use example/multi-pods, as cross-platform build is very slow when running, go build ..., we should make project source code as simple as possible when using cross-platform/multi-platform builds.
```
example/multimicroservices doesn't actually use `cross-platform/multi-platform` builds.  I'm not sure this needs to be changed, was this actually slowing things down?  This doesn't actually use cross-platform/multi-platform builds